### PR TITLE
switched from firebase to pouchdb

### PR DESCRIPTION
**Pull request by eibrahim**
I created a serializer to handle parsing of githubData attribute but I think, the model should be changed so that the serializer can just set those porperties directory on the model, so instead of 

`issue.githubData.state`

it would be

`issue.state`

If you are cool with that, I can issue another PR

### CODE REVIEW

**Comment by acorncom** [comment]
@eibrahim Yes, let's move that data out of `githubData`, that was mainly done to keep Firebase happy.  Could we do that on this PR?

**Comment by eibrahim** [comment]
I can do it in this PR.  stay tuned.

**Comment by eibrahim** [comment]
how do I add to an existing PR?

File: app/serializers/issue.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    json.githubData = githubDataAsString;
+    return json;
+  },
+  normalize(typeClass, hash) {
+    let o = this._super(...arguments);
+    let githubDataAsObject = o.data.attributes.githubData;
+    o.data.attributes.githubData = JSON.parse(githubDataAsObject);
+    return o;
+  }
+ */
```
Comment (acorncom) [inline app/serializers/issue.js]: Should we take these lines out?

**Comment by eibrahim** [comment]
Yes, I should probably just delete the serializer since i am not using it anymore.

**Pull request merged**